src/esp/sim/ClassicReplayRenderer.cpp

```diff
@@ -236,11 +236,14 @@ void ClassicReplayRenderer::doRender(
       }
 
 #ifdef ESP_BUILD_WITH_BACKGROUND_RENDERER
+      esp::gfx::RenderCamera::Flags flags{};
+      if (config_.enableFrustumCulling) {
+        flags |= gfx::RenderCamera::Flag::FrustumCulling;
+      }
```

Comment:
Probably want to change this so that if disableFrustumCulling is true then it overrides whatever the config specifies.

Reply:
I don't understand this comment. There's `config_.ourNewCullingFlag` (name TBD). Is there also another config that is relevant here?

Reply:
From what I understand this PR is providing a RenderReplay-specific flag to disable culling if necessary, regardless of SimulatorConfiguration setting, but perhaps my understanding is wrong.

Reply:
After discussing with Mikael, I see that SimConfig's role in this process with ReplayRenderer is not as relevant as I had initially thought (not relevant at all, in fact :) ), so ignore my suggestions.